src/coreclr/System.Private.CoreLib/src/System/GC.CoreCLR.cs

```diff
@@ -614,6 +614,59 @@ internal static void RegisterMemoryLoadChangeNotification(float lowMemoryPercent
             }
         }
 
+        [Flags]
+        internal enum NoGCRegionCallbackWork
```

Comment:
> internal enum NoGCRegionCallbackWork [](http://example.com/codeflow?start=7&length=37)

nit - how about call this NoGCRegionCallbackAction instead of NoGCRegionCallbackWork?